src/org/usfirst/frc/team467/robot/TalonSpeedControllerGroup.java

```diff
@@ -187,4 +208,23 @@ public int sensorPosition() {
 		}
 		return leader.getSelectedSensorPosition(0);
 	}
+	
+	/**
+	 * Checks for the amount of Motors that you made
+	 * @return
+	 */
+	public int motorAmt(){
+		if(leader != null) {
+			return 1;
+			}
+		if(follower1 != null) { 
+			return 2;
+			}
+		if(follower2 != null) {
+			return 3;
+		}
+		else { 
+			return 0;
+			}
+	}
```

Comment:
The logic of this function is wrong. Is it ever called? If not, just remove it. Otherwise, it needs to be fixed.